### ISSUE TITLE
Default for the stage release script should not be the current user

### ISSUE THREAD

**Issue by mattcaswell** [openssl/tools#167]
When running stage-release.sh if you neglect to supply the "--local-user" argument then it defaults to the current user. This is almost never the correct answer since it signs the release tarball with the current user's gpg key, rather than the team key.

It probably should default to the team key, or alternatively fail.

**Comment by levitte** [comment]
I would default it to the team key.  I just found out that the key user id (i.e. any email address associated with it) can be used in place of the hexadecimal id, so it would be a good idea to use that as a default.

**Comment by levitte** [comment]
I have a few things lying around that align with this.  I'll collect them into a PR on Monday.

**Comment by levitte** [comment]
@quarckster, this is pretty irrelevant nowadays, yeah?

**Comment by quarckster** [comment]
Yes, stage-release.sh is significantly cleaned up in https://github.com/openssl/tools/pull/240 and it does not sign anything at all.
